cardinal_service_profile.profile

```diff
@@ -17,6 +17,19 @@ use Drupal\Core\Site\Settings;
 use Drupal\menu_link_content\Entity\MenuLinkContent;
 use Drupal\node\NodeInterface;
 
+/**
+ * Migration callback to just get the current timestamp.
+ *
+ * @param mixed $arg
+ *   Passed parameter from migration plugin `callback`.
+ *
+ * @return int
+ *   Current timestamp.
+ */
+function cardinal_service_profile_get_time($arg = NULL) {
```

Comment:
Where is this used? Why not just use `time()`?

Reply:
thats what you'd think right?! but nope. when you use `time()` it logs messages like "Function accepts 0 arguments but 1 passed" something like that.

Reply:
Where is this being used? PhpStorm isn't giving me anything.

Reply:
https://github.com/SU-HKKU/cardinal_service_profile/pull/6/files#diff-dad26c2dac86ef73ece9ba303bf1e20dR119

Reply:
Oof.

Reply:
I'll defer to your hard-won experience on this; no need for more dents in my forehead.

Please comment this function extensively based on [your comments here](https://github.com/SU-HKKU/cardinal_service_profile/pull/6#discussion_r410517825). It's clearly a confusing function.

Reply:
I added comments on the docblock for this function